File: AutoIt.Foundation/AutoIt.Foundation.Common/LangAnaly/Model/Produce/Group.cs
using System.Collections.Generic;

namespace AutoIt.Foundation.Common.LangAnaly.Model
{
    public class Group:EgtEntityBase
    {
        public string Name { get; set; }
        public Symbol Container { get; set; }
        public Symbol Start { get; set; }
        public Symbol End { get; set; }
        public AdvanceMode AdvanceMode { get; set; }
        public EndingMode EndingMode { get; set; }
        public List<Symbol> NestGroup { get; set; } = new List<Symbol>();
    }
}

File: AutoIt.Foundation/AutoIt.Foundation.Web.Scripts/CodeEdit/LangAnaly/Model/Group.ts
module CodeEdit.LangAnaly.Model {
    //·Ö×é
    export class Group extends EgtEntityBase{
        Name: string;
        Container: CodeEdit.LangAnaly.Model.Symbol;
        Start: CodeEdit.LangAnaly.Model.Symbol;
        End: CodeEdit.LangAnaly.Model.Symbol;
        AdvanceMode: CodeEdit.LangAnaly.Model.AdvanceMode;
        EndingMode: CodeEdit.LangAnaly.Model.EndingMode;
        NestGroup: List<CodeEdit.LangAnaly.Model.Symbol>;
    }
}